ykcompile/src/lib.rs

```diff
@@ -96,7 +96,6 @@ impl<TT> CompiledTrace<TT> {
 
     /// Actually call the code. This is a separate unmangled function to make it easy to set a
     /// debugger breakpoint right before entering the trace.
-    #[no_mangle]
```

Comment:
Kill the comment above too.

Reply:
Now that it's mangled, do we even still need this function?

Reply:
Can you try the rust-gdb thing and see if you can get it to break on the mangled name?

If that works, we should leave this as a func.